lib/vsc/accounting/elasticsearch.py

```diff
@@ -176,3 +176,14 @@ def set_source(self, extra=None):
 
         self.search = self.search.source(self.fields)
         self.log.debug("ES query [%s] retrieving fields: %s", self.id, ','.join(self.fields))
+
+    def scan_hits(self):
+        """
+        Scan all hits in Search object and handle any errors
+        """
+        try:
+            hits = [hit for hit in self.search.scan()]
```

Comment:
isn't this very inefficient? You do a list here, return it and then you do a again a list scan with `to_dict`. Can't you do both in once go?

Reply:
good point, fixed in https://github.com/vub-hpc/vsc-accounting-brussel/pull/4/commits/5fff34fcd9eb3e8ec9cf7d6f506c5a3aedb76052